modules/setting/setting.go

```diff
@@ -931,6 +932,8 @@ func NewContext() {
 	sec = Cfg.Section("U2F")
 	U2F.TrustedFacets, _ = shellquote.Split(sec.Key("TRUSTED_FACETS").MustString(strings.TrimRight(AppURL, "/")))
 	U2F.AppID = sec.Key("APP_ID").MustString(strings.TrimRight(AppURL, "/"))
+
+	zip.Verbose = false
```

Comment:
Do we still need this? I'm not sure we use cae/zip elsewhere.

Reply:
Don't know. Since this is a refactor PR, I just keep it.